ui-automation-tests/exampleotp-features/register.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Users can sign up to the self-service experience using real data.
+
+  Background:
+    Given the user is on the "/register" page
+
+  Rule: Register Account
+    Scenario: The user registers a new account
+      When they submit the email "~Registered"
```

Comment:
If feature is enabled in prod by mistake and they provide '~Registered' then they will be able to create account without any verification.

Reply:
Yes you are quite correct but that is also true if we simply had fixed Email Addresses and Fixed OTP Codes etc.. 

I am a bit confused since we have discussed all this previously and agreed a solution which you are now concerned about. 

Any fixed OTP System will, if allowed to run in Production, will allow free reign that is why we have the Flag USE_STUB_OTP to prevent this and, to my mind, is no different to the approach adopted by other teams whereby they use similar flags; Please can I refer you to other teams approach where they have a TEST_CLIENTS_ENABLED Flag (so same thing but different name) which is only set in the Environments where they wish the functionality to work (please see https://govukverify.atlassian.net/wiki/spaces/LO/pages/3503128761/Fixed+One+Time+Password+OTP+for+test+users)